server/src/main/java/org/elasticsearch/action/bulk/TransportBulkAction.java

```diff
@@ -809,8 +866,9 @@ private void processBulkIndexIngestRequest(
     ) {
         final long ingestStartTimeInNanos = System.nanoTime();
         final BulkRequestModifier bulkRequestModifier = new BulkRequestModifier(original);
-        ingestService.executeBulkRequest(
-            original.numberOfActions(),
+        int numberOfActions = original.numberOfActions();
+        getIngestService(original).executeBulkRequest(
+            numberOfActions,
```

Comment:
I think this is a leftover change? It's not a big deal to leave in, just curious if it was for debugging or something

Reply:
I have absolutely no idea. I just reverted it.